support-frontend/assets/helpers/tracking/acquisitions.ts

```diff
@@ -36,33 +36,33 @@ export type OphanIds = {
 // https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js
 export type ReferrerAcquisitionData = {
 	campaignCode?: string;
-	referrerPageviewId: string | null | undefined;
+	referrerPageviewId?: string | null;
 	referrerUrl?: string;
-	componentId: string | null | undefined;
-	componentType: string | null | undefined;
-	source: string | null | undefined;
-	abTests: AcquisitionABTest[] | null | undefined;
+	componentId?: string | null;
+	componentType?: string | null;
+	source?: string | null;
+	abTests?: AcquisitionABTest[] | null;
 	// these aren't in the referrer acquisition data model on frontend, but they're convenient to include
 	// as we want to include query parameters in the acquisition event to e.g. facilitate off-platform tracking
-	queryParameters: AcquisitionQueryParameters | null | undefined;
-	labels: string[] | null | undefined;
+	queryParameters?: AcquisitionQueryParameters | null;
+	labels?: string[] | null;
 };
 
 export type PaymentAPIAcquisitionData = {
 	pageviewId: string;
-	visitId: string | null | undefined;
-	browserId: string | null | undefined;
-	platform: string | null | undefined;
-	referrerPageviewId: string | null | undefined;
-	referrerUrl: string | null | undefined;
-	campaignCodes: string[] | null | undefined;
-	componentId: string | null | undefined;
-	componentType: string | null | undefined;
-	source: string | null | undefined;
-	abTests: AcquisitionABTest[] | null | undefined;
-	gaId: string | null | undefined;
-	queryParameters: AcquisitionQueryParameters | null | undefined;
-	labels: string[] | null | undefined;
+	visitId?: string | null;
```

Comment:
Do we still need all the nulls here? Could we restrict it to `?: string`?

Reply:
I'm not too sure, I'd need to go on a bit of a hunt in the codebase for anywhere that we may explicitly be setting one of these values to null

Reply:
We could get rid a fair few of the nulls here 👍, we only need a few for props that call a shared helper function